### DISCUSSION TITLE
Can't config anything

### DISCUSSION THREAD

**Discussion by samuel-casasdray** [tailwindlabs/tailwindcss#13311] [Help]
So, I'm starting tailwind as a small scale project and as I'm programming I needed a smaller screen size case. 
I tried to place it in my tailwind.config.js, but I can never use it…
I reproduce my config in the playground, but I can't make it to work… https://play.tailwindcss.com/ObtPeFPWLF?file=config
I also try to config a new style without success.

(I used create-react-app, and so I use React and material-tailwind)

Here is the code : 
```/** @type {import('tailwindcss').Config} */
const withMT = require("@material-tailwind/react/utils/withMT");
const plugin = require("tailwindcss");
module.exports = withMT({
	content: ["./src/**/*.{html,js,jsx,ts,tsx}"],
	theme: {
		screen: {
			"sm": "640px",
			"md": "768px",
			"lg": "1024px",
			"xl": "1280px",
			"2xl": "1536px",
		},
        extends: {
            screen: {
                "xsm": "425px",
            }
        }
	},
	plugins: [
		plugin(function ({ matchUtilities }) {
			matchUtilities(
				{
					"grid-cols-repeat": (value) => {
						return {
							"grid-template-columns":
								"repeat(auto-fill, " + value + "px)",
						};
					},
				},
				{ values: { 300: "300", 400: "400" } }
			);
		}),
	],
});
```

**Comment by maxlshk** [comment]
There's a typo in your Tailwind CSS configuration. The correct way to extend the default theme is by using `extend` instead of `extends`. Additionally, the correct property for defining screen sizes is `screens`, not `screen`.

```
...
extend: {
      screens: {
        xsm: '425px',
      },
    },
...
```

Here's your corrected TailwindPlay: https://play.tailwindcss.com/7QayMUXgLu?file=config
